priority_variables_transform/CHS-ingest/hrtrt.yaml

```diff
@@ -518,6 +554,9 @@
         associated_visit:
           value: CHS YEAR 3
           range: string
+        age_at_observation:
+          expr: ({pht001452.phv00100487} * 365) + 1095
+          lookup_key: phv00100285          
```

Comment:
Hmm, I don't think I know what this is either... We will probably need to do something like use a lookup key but for now I don't think this is supported.

Reply:
I will delete.